TWTValidation/JSON Validator/Processors/TWTJSONObjectValidatorGenerator.m

```diff
@@ -214,7 +211,14 @@ - (void)processBooleanValueNode:(TWTJSONSchemaBooleanValueASTNode *)booleanValue
 {
     BOOL validates = booleanValueNode.booleanValue;
     [self pushNewObject:[[TWTBlockValidator alloc] initWithBlock:^BOOL(id value, NSError *__autoreleasing *outError) {
-        // Error needs to be written in validator class because context is unknown here
+        if (!validates && outError) {
+            // If in the future a booleanValue node is used for any node properties other than additional items/properties,
+            // the error code must be updated
+            *outError = [NSError twt_validationErrorWithCode:TWTValidationErrorCodeAdditionalElementsNotAllowed
+                                            failingValidator:nil
+                                                       value:nil
+                                        localizedDescription:TWTLocalizedString(@"TWTJSONObjectValidator.validationError")];
```

Comment:
We really sure include the failing validator and value here. You’ll need to create a block variable for the block validator and refer to it inside the block.